Fix an error with empty galleries.

madbonkey opened the pull request:
The + operand produces a fatal error with an empty $thumbnails array. Added a quick fix.

madbonkey committed:
Fix an error with empty galleries.

The + operand produces a fatal error with an empty $thumbnails array. Added a quick fix.

madbonkey committed:
Fix indentation.

ccschmitz commented:
@NicolasSchneider You need to update the spacing on your conditionals still.

```
if( empty($thumbnails) )
```

needs to be:

```
if (empty($thumbnails))
```

madbonkey committed:
Update spacing on the conditional …

… although the necessity is unclear to me, a few lines down the spacing
is done the other way again.
